mypy/test/data/check-functions.test

```diff
@@ -1161,6 +1161,45 @@ f(g1)
 f(g2)
 f(g3)
 
+-- (...) -> T
+-- ----------------
+[case testEllipsisWithArbitraryArgsOnBareFunction]
+def f(x, y, z): # type: (...) -> None
```

Comment:
I'd add a couple things to these (non-error) tests:
- a call to the function that shows that you can actually pass it arguments
- arguments with defaults and kwarg-only args